etl/src/energuide/embedded/heating.py

```diff
@@ -61,6 +62,8 @@ class Heating(_Heating):
         4: EnergySource.PROPANE,
         5: EnergySource.WOOD,
         6: EnergySource.WOOD,
+        7: EnergySource.WOOD,
+        8: EnergySource.WOOD,
```

Comment:
▶️ Glad to see we're moving four-wood with this. ▶️

Reply:
:neutral_face: